backend/helper_functions.py

```diff
@@ -325,3 +326,13 @@ def user_validation(data):
         )
 
     return None, status.HTTP_200_OK
+
+
+def generate_unique_id():
+    """
+    Generate a new unique ID.
+
+    Returns:
+        str: A new unique ID.
+    """
+    return str(uuid.uuid4())
```

Comment:
I like the addition of this generate_unique_id helper function. This will allow us to create unique id's for the comments so that id's don't overlap!